lib/galaxy/webapps/galaxy/api/configuration.py

```diff
@@ -20,6 +21,19 @@ def __init__( self, app ):
         super( ConfigurationController, self ).__init__( app )
         self.config_serializer = configuration.ConfigSerializer( app )
         self.admin_config_serializer = configuration.AdminConfigSerializer( app )
+        self.user_manager = users.UserManager( app )
+
+    @expose_api_anonymous
+    def whoami( self, trans, **kwd ):
+        """
+        GET /api/authenticate/whoami
```

Comment:
Fixed. I feel a bit like we shouldn't include that and instead have it auto-constructed in the docs?